workers/unity/Packages/io.improbable.gdk.core/Tests/Editmode/Components/OptionTests.cs

```diff
@@ -179,9 +146,9 @@ public void Implicit_conversion_to_payload_type_works()
         public void Implicit_conversion_to_payload_type_using_empty_option_throws()
         {
             var option = new Option<bool>();
-            Assert.Throws<CalledValueOnEmptyOptionException>(() =>
+            Assert.Throws<EmptyOptionException>(() =>
             {
-                var _ = option.Value;
+                bool _ = option.Value;
```

Comment:
The test name and content don't really match up.

I'd expect something like: 

```
bool _ = option;
```

to actually test the `Option<T> -> T` implicit conversion

Reply:
Knew I forgot something, thanks!